lib/get_user.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+function getUser(token) {
+  var data = token.split('.')[1];
+  data = data.replace(/-/g, '+').replace(/_/g, '/');
+
+  var mod = data.length % 4;
+  if (mod === 2) data += '==';
+  if (mod === 3) data += '=';
+  if (mod === 1 || mod > 3) return null;
+
+  try {
+    data = (new Buffer(data, 'base64')).toString('utf8');
+    return JSON.parse(data).u;
+  } catch(err) {
+    return null;
+  }
+}
+
+module.exports = getUser;
```

Comment:
When I looked for modules I wasn't aware that browserify does some polyfilling, so I was trying to find something that was node + browser. The client side libs for this I found are **way** overkill for what ends up being incredibly straightforward code. I see no reason to bring in a dependency to do this.